app/scripts/modules/netflix/fastProperties/scope/fastPropertyScopeSearchCategory.service.ts

```diff
@@ -103,7 +103,7 @@ export class FastPropertyScopeCategoryService {
   public impactCountForRegions(scope: Scope): ng.IPromise<IImpactCounts> {
     return this.fastPropertyReader.fetchImpactCountForScope(scope)
       .then((results: any) => {
-        return <IImpactCounts>{total: results.count, up: results.count};
+        return <IImpactCounts>{total: parseInt(results.count, 10), up: parseInt(results.count, 10)};
```

Comment:
I think we can disable the radix linter warning for this as base 10 is the default so if you wanna drop it here, that's cool.  You'll need to set `"radix": false` in `tslint.json`.